plugins/utils/competitest.nix

```diff
@@ -0,0 +1,202 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "competitest";
+  originalName = "competitest.nvim";
+  defaultPackage = pkgs.vimPlugins.competitest-nvim;
+
+  maintainers = [helpers.maintainers.svl];
+
+  extraOptions = with lib.types; {
+    extraConfig = helpers.mkNullOrOption (attrsOf anything) ''
+      Extra config options for competitest.nvim
+    '';
+  };
+
+  settingsOptions = {
+    local_config_file_name = helpers.defaultNullOpts.mkStr ".competitest.lua" ''
+      you can use a different configuration for every different folder. See
+      [local configuration](https://github.com/xeluxee/competitest.nvim?tab=readme-ov-file#local-configuration)
```

Comment:
```suggestion
      You can use a different configuration for every different folder.
      See configuration](https://github.com/xeluxee/competitest.nvim?tab=readme-ov-file#local-configuration)
```

All descriptions should start with a capital letter.
Prefer breaking lines after periods (`.`).

Sorry for annoying you with this kind of detail but we try to keep consistence across the documentation.